scalableminds/webknossos
# Reactify Project List View

I took issue #1431 as an occasion to reactify the project list admin view. 

### Mailable description of changes:
- Updated Project List Admin View with more search and sorting options
- The "Delete" action is only displayed for the project owner to avoid confusion with the required permissions.

### Steps to test:
- Go to `/projects`
- Create a few projects
- Check that:
  - search is working
  - sorting is working
  - deletion is working


1. Create a new second user. 
2. Make him admin in the same project as one of the projects. 
3. Navigate to `/projects`
4. Notice that "delete" action is not display, since the new user is not the owner of the project

### Issues:
- fixes #1431 

------
